tools/flexynesis/macros.xml

```diff
@@ -0,0 +1,56 @@
+<macros>
+    <token name="@TOOL_VERSION@">0.2.10</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <token name="@PROFILE@">24.1</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION@">flexynesis</requirement>
+            <yield/>
+        </requirements>
+    </xml>
+    <xml name="edam">
+        <edam_topics>
+            <edam_topic>topic_0622</edam_topic>
+            <edam_topic>topic_3474</edam_topic>
+            <edam_topic>topic_2640</edam_topic>
+        </edam_topics>
+        <edam_operations>
+            <edam_operation>operation_3197</edam_operation>
+            <edam_operation>operation_2403</edam_operation>
+            <edam_operation>operation_2426</edam_operation>
+        </edam_operations>
+    </xml>
+    <token name="@CHECK_NON_COMMERCIAL_USE@"><![CDATA[
+        #if not $non_commercial_use
+            >&2 echo "this tool is only available for non commercial use";
+            exit 1;
+        #end if
+    ]]></token>
+    <xml name="advanced">
+        <section name="advanced" title="Advanced Options">
+            <param argument="--fusion_type" type="select" label="Fusion method" help="How to fuse the omics layers?">
+                <option value="intermediate">intermediate</option>
+                <option value="early">early</option>
+            </param>
+            <param argument="--finetuning_samples" type="integer" min="0" value="0" label="fine-tuning" help="Number of samples from the test dataset to use for fine-tuning the model. Set to 0 to disable fine-tuning." />
```

Comment:
```suggestion
            <param argument="--finetuning_samples" type="integer" min="0" value="0" label="Number of samples from the test dataset to use for fine-tuning the model." help="Set to 0 to disable fine-tuning." />
```

Reply:
the help text sounds like a better label for the params. For eg. "fine-tuning" does not mean anything for the users. Please change the remaining param labeles too.